[2022-04-28 01:07:27,090] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.download_crash_dataset_task scheduled__2022-04-28T00:00:00+00:00 [queued]>
[2022-04-28 01:07:27,132] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.download_crash_dataset_task scheduled__2022-04-28T00:00:00+00:00 [queued]>
[2022-04-28 01:07:27,134] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 01:07:27,135] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-04-28 01:07:27,136] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 01:07:27,181] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_crash_dataset_task> on 2022-04-28 00:00:00+00:00
[2022-04-28 01:07:27,202] {standard_task_runner.py:52} INFO - Started process 149 to run task
[2022-04-28 01:07:27,231] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'local_2_gcs_2_bq_dag', 'download_crash_dataset_task', 'scheduled__2022-04-28T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/download_2_gcs_dag_V2.py', '--cfg-path', '/tmp/tmp4p90fu3b', '--error-file', '/tmp/tmpyyylih18']
[2022-04-28 01:07:27,240] {standard_task_runner.py:77} INFO - Job 4: Subtask download_crash_dataset_task
[2022-04-28 01:07:27,574] {logging_mixin.py:109} INFO - Running <TaskInstance: local_2_gcs_2_bq_dag.download_crash_dataset_task scheduled__2022-04-28T00:00:00+00:00 [running]> on host 130e3b0cefbc
[2022-04-28 01:07:27,960] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_2_gcs_2_bq_dag
AIRFLOW_CTX_TASK_ID=download_crash_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-28T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-28T00:00:00+00:00
[2022-04-28 01:07:27,963] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-28 01:07:27,972] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://data.cityofchicago.org/api/views/85ca-t3if/rows.csv?accessType=DOWNLOAD > /opt/***/85ca-t3if.csv']
[2022-04-28 01:07:28,099] {subprocess.py:85} INFO - Output:
[2022-04-28 01:08:21,507] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-28 01:08:22,002] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=local_2_gcs_2_bq_dag, task_id=download_crash_dataset_task, execution_date=20220428T000000, start_date=20220428T010727, end_date=20220428T010822
[2022-04-28 01:08:22,453] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-28 01:08:23,103] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-04-28 01:54:36,936] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.download_crash_dataset_task scheduled__2022-04-28T00:00:00+00:00 [queued]>
[2022-04-28 01:54:36,961] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.download_crash_dataset_task scheduled__2022-04-28T00:00:00+00:00 [queued]>
[2022-04-28 01:54:36,963] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 01:54:36,964] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-04-28 01:54:36,966] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 01:54:36,998] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_crash_dataset_task> on 2022-04-28 00:00:00+00:00
[2022-04-28 01:54:37,018] {standard_task_runner.py:52} INFO - Started process 516 to run task
[2022-04-28 01:54:37,046] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'local_2_gcs_2_bq_dag', 'download_crash_dataset_task', 'scheduled__2022-04-28T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/download_2_gcs_dag_V2.py', '--cfg-path', '/tmp/tmp3duuc0vn', '--error-file', '/tmp/tmpdbjd7hz6']
[2022-04-28 01:54:37,052] {standard_task_runner.py:77} INFO - Job 3: Subtask download_crash_dataset_task
[2022-04-28 01:54:37,222] {logging_mixin.py:109} INFO - Running <TaskInstance: local_2_gcs_2_bq_dag.download_crash_dataset_task scheduled__2022-04-28T00:00:00+00:00 [running]> on host 1d2c8c1aba91
[2022-04-28 01:54:37,405] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_2_gcs_2_bq_dag
AIRFLOW_CTX_TASK_ID=download_crash_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-28T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-28T00:00:00+00:00
[2022-04-28 01:54:37,417] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-28 01:54:37,420] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://data.cityofchicago.org/api/views/85ca-t3if/rows.csv?accessType=DOWNLOAD > /opt/***/85ca-t3if.csv']
[2022-04-28 01:54:37,492] {subprocess.py:85} INFO - Output:
[2022-04-28 01:55:30,361] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-28 01:55:30,596] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=local_2_gcs_2_bq_dag, task_id=download_crash_dataset_task, execution_date=20220428T000000, start_date=20220428T015436, end_date=20220428T015530
[2022-04-28 01:55:30,873] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-28 01:55:31,493] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
